src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2446,6 +2446,16 @@ public static void UtcAliases_MapToUtc()
             }
         }
 
+        [Fact]
+        public static void UtcAliases_AreEqualToUtc()
```

Comment:
>public static void UtcAliases_AreEqualToUtc() [](http://example.com/codeflow?start=8&length=45)

need to use `[PlatformSpecific(~TestPlatforms.Windows & ~TestPlatforms.Browser)]`?

Reply:
As I linked in the description, `GetAdjustmentRules()` on windows also has logic to return empty array when the  underlying is null. This test is relevant (and should pass) on Windows

Reply:
It is possible to run on Windows machines that do not have ICU library. At that time, you cannot create the time zone objects listed in `s_UtcAliases`.

Reply:
maybe we can do
``` csharp
foreach (string alias in s_UtcAliases)
{
    try
    {
        TimeZoneInfo actualUtc = TimeZoneInfo.FindSystemTimeZoneById(alias);
        // ...
    }
    catch (TimeZoneNotFoundException) { }
}
```

so if the timezones can be found, they are tested

Reply:
This is what the test data in `SystemTimeZonesTestData()` does
https://github.com/dotnet/runtime/blob/c88dbd3f943b8206b4821aaf10ddf3c216100942/src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs#L2322-L2330

Reply:
It is better to use `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsIcuGlobalization))]` on the method.